pyproject.toml

```diff
@@ -23,6 +23,7 @@ dependencies = [
     "python-dateutil>=2.8.2,<3",
     "rich>=13.5.2,<14",
     "pydantic>=2.3.0,<3",
+    "PyMuPDF>=1.24.10,<2",
```

Comment:
Can you remove this line?
- I can't put PyMuPDF in default dependencies due to licensing issues.
- Also, there are issues with PyMuPDF>=1.25 with other parsers (CAMS and KFINTECH) . I'm working on resolving this .